src/playlist.js

```diff
@@ -308,6 +308,40 @@ const calculateExpiredTime = function(playlist, expiredSync, segmentSync) {
 };
 
 /**
+ * Returns the end point the the playlist
+ *
+ * @param {Object} playlist a media playlist object
+ * @returns {Object} an object containing the time length of the playlist
+ * @function playlistEnd
+ */
+
+export const playlistEnd = function(playlist) {
```

Comment:
this function is very similar to `seekable` defined just below. Is there any way we can reduce the amount of repeat code here?